ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/BlockContainer.java

```diff
@@ -13,4 +13,16 @@
 
 package tech.pegasys.teku.spec.datastructures.blocks;
 
-public interface BlockContainer {}
+import java.util.Optional;
+import tech.pegasys.teku.spec.datastructures.blobs.versions.deneb.SignedBlobSidecars;
+
+public interface BlockContainer {
+
+  default Optional<SignedBeaconBlock> getSignedBeaconBlock() {
```

Comment:
I am thinking this shouldn't be an Optional. It will always be there regardless of the returned type right?

Reply:
Yep I thought about that. The thing is it will clash with the `BlockSummary` `getSignedBeaconBlock()` otherwise

Reply:
Yeah, I get it, let's leave it like this for now.